### PR TITLE
Feature/create fluentbit chart

### PR DESCRIPTION
**Pull request by rddmoreira**

### CODE REVIEW

File: charts/fluent-bit/templates/daemonset.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      containers:
+        - name: service
+          securityContext:
+{{ toYaml $values.securityContext | indent 12 }}
+          env:
+          image: {{ printf "%s/%s:%s" $values.image.registry $values.image.repository $values.image.tag }}
+          imagePullPolicy: {{ default $values.image.pullPolicy "IfNotPresent" }}
+          ports:
+  {{- if and $values.prometheus.enabled }}
+            - name: metrics
```
Comment (pmrocha-mindera) [inline charts/fluent-bit/templates/daemonset.yaml:49]: If the purpose of exposing this container port is only for metric collection , we don't need to expose it.

File: charts/fluent-bit/values.yaml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+nameOverride: "fluent-bit"  # replace this by your service name
+fullnameOverride: ""  # fill if you whish to override the generatedChartFullName
+name: &nameOverride
+
+# Controls if prometheus scrapping should be enabled for this service
+prometheus:
+  enabled: true  # enables/disable scraping of prometheus metrics
+  path: "/api/v1/metrics/prometheus"  # path for scrapping prometheus metrics
+  port: 2020  # port were the metrics web server is running
+  scheme: "tcp"  # tcp connection scheme
```
Comment (pmrocha-mindera) [inline charts/fluent-bit/values.yaml]: tcp scheme is not supported only http or https

File: charts/fluent-bit/values.yaml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+image:
+  registry: docker.io  # container registry url
+  repository: amazon/aws-for-fluent-bit  # container registry repo name
+  tag: 2.3.0  # image tag
+  pullPolicy: Always  # kubernetes container image pull policy
+
+# limits the container resources
+resources:
+  # resources that the container requires to boot
+  requests:
```
Comment (pmrocha-mindera) [inline charts/fluent-bit/values.yaml:47]: are you sure that it needs this much?

**Review by pmrocha-mindera — APPROVED** [review]

File: charts/fluent-bit/values.yaml (hunk 3 above)
Comment (rddmoreira) [reply charts/fluent-bit/values.yaml:47]: Saw that values in some random config. I will review the official config and copy the values from there.

File: charts/fluent-bit/values.yaml (hunk 2 above)
Comment (rddmoreira) [reply charts/fluent-bit/values.yaml]: I will update to http

**Review by pedro-evangelista — APPROVED** [review]

**Pull request merged**